go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/salesforce/kubelet-summary-exporter
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```

Comment:
what's the motivation behind pinning a toolchain? do we need this, or do we at least want to allow upgrades?